source/common/upstream/cluster_manager_impl.h

```diff
@@ -364,6 +364,11 @@ class ClusterManagerImpl : public ClusterManager,
   // Upstream::MissingClusterNotifier
   void notifyMissingCluster(absl::string_view name) override;
 
+  std::shared_ptr<const envoy::config::cluster::v3::Cluster::CommonLbConfig>
+  GetCommonLbConfigPtr(const envoy::config::cluster::v3::Cluster::CommonLbConfig common_lb_config) override {
```

Comment:
nit: pass by reference instead of value.

Reply:
Done